src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/AuthorizationFilterContext.cs

```diff
@@ -19,14 +19,34 @@ public class AuthorizationFilterContext : FilterContext
         public AuthorizationFilterContext(
             ActionContext actionContext,
             IList<IFilterMetadata> filters)
+            : this(actionContext, filters, combineAuthorizeFilters: false)
+        {
+        }
+
+        /// <summary>
+        /// Instantiates a new <see cref="AuthorizationFilterContext"/> instance.
+        /// </summary>
+        /// <param name="actionContext">The <see cref="ActionContext"/>.</param>
+        /// <param name="filters">All applicable <see cref="IFilterMetadata"/> implementations.</param>
+        /// <param name="combineAuthorizeFilters">The <see cref="CombineAuthorizeFilters"/>.</param>
+        public AuthorizationFilterContext(
+            ActionContext actionContext,
+            IList<IFilterMetadata> filters,
+            bool combineAuthorizeFilters)
```

Comment:
This isn't the right level for an option like this to appear.

Reply:
What I mean to say is that this is much too visible of a place for this to show up, things in this package live forever.